src/main/java/seedu/mindmymoney/command/CalculateCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.mindmymoney.command;
+
+import seedu.mindmymoney.MindMyMoneyException;
+import seedu.mindmymoney.data.ExpenditureList;
+import seedu.mindmymoney.helper.GeneralFunctions;
+
+import static seedu.mindmymoney.constants.Indexes.INDEX_OF_FIRST_ITEM_IN_STRING;
+import static seedu.mindmymoney.constants.Flags.FLAG_OF_EXPENDITURE_PER_MONTH;
+import static seedu.mindmymoney.constants.Indexes.INDEX_OF_SECOND_ITEM_IN_STRING;
+import static seedu.mindmymoney.helper.Calculations.calculateExpenditurePerMonth;
+
+/**
+ * Represents the Calculate command.
+ */
+public class CalculateCommand extends Command {
```

Comment:
Great implementation! Perhaps it would be better if we name this class more specifically e.g CalculateInputCommand as in the future if we have different things we would need to calculate it would be well differentiated.

Reply:
ok!